### PR TITLE
Support accessibility labels on iOS switches.

### PR DESCRIPTION
**Pull request by brandondiamond**
This is the engine change needed to support flutter/flutter#39993.

Queries / tracks the accessibility in iOS and plumbs it through to the framework via AccessibilityFeatures.

Issue: flutter/flutter#4830.

### CODE REVIEW

**Comment by jonahwilliams** [comment]
@brandondiamond I think you might be synced to a bad PR, can you pull an updated master into your PR?

**Comment by jonahwilliams** [comment]
Also, please check the errors due to code formatting

**Comment by brandondiamond** [comment]
Fixed formatting and merged with master (hopefully my git-fu didn't mess everything up)!

**Comment by brandondiamond** [comment]
Missed a change in web_ui, should be fixed and ready for review :)

Thanks again, Jonah.

**Comment by chinmaygarde** [comment]
This commit is causing [iOS LUCI failures](https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket.appspot.com/8901230881953480640/+/steps/build_ios_debug_sim/0/stdout).

**Comment by chinmaygarde** [comment]
UIAccessibilityIsOnOffSwitchLabelsEnabled is only available when using the iOS 13 SDK. Buildbots have not migrated to that yet.

File: shell/platform/darwin/ios/framework/Source/FlutterViewController.mm [hunk 1]
```diff
@@ -234,6 +234,16 @@ - (void)setupNotificationCenterObservers {
...
 
+  [center addObserver:self
+             selector:@selector(onAccessibilityStatusChanged:)
+                 name:UIAccessibilityOnOffSwitchLabelsDidChangeNotification
+               object:nil];
+
+  [center addObserver:self
+             selector:@selector(onAccessibilityStatusChanged:)
+                 name:UIAccessibilityOnOffSwitchLabelsDidChangeNotification
+               object:nil];
```
Comment (cbracken) [inline shell/platform/darwin/ios/framework/Source/FlutterViewController.mm]: When this is re-landed, only one of these two duplicate notifications should be applied.

**Pull request merged**